src/main/java/com/school/project/controller/UserController.java

```diff
@@ -1,9 +1,60 @@
 package com.school.project.controller;
 
-import org.springframework.web.bind.annotation.RestController;
+import com.school.project.dto.UserDto;
+import com.school.project.model.User;
+import com.school.project.repository.UserRepository;
+import com.school.project.service.UserService;
+import org.modelmapper.ModelMapper;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+import java.util.stream.Collectors;
 
 @RestController
+@RequestMapping
 public class UserController {
+    @Autowired
+    ModelMapper modelMapper;
+    @Autowired
+    UserService userService;
+    @Autowired
+    UserRepository userRepository;
+
+    @PostMapping("/users")
```

Comment:
This class is already in another (still open) pull request. As long as another one is changed and merged, you will get a merge conflict.